model/text_proto_assets.bzl

```diff
@@ -119,15 +119,16 @@ def _generate_single_asset_proto_binary(
     Args:
         name: str. The name of this target.
         proto_file_name: str. The file name of the text proto under the assets directory that will
-            be converted. This is assuming to correspond to 'src/main/assets/<name>.textproto' and
-            will lead to a new generated file called 'src/main/assets/<name>.pb'.
-        proto_dep_name: str. The name of the proto library under //model that contains the proto
+            be converted. This is assuming to correspond to '<asset_dir>/<name>.textproto' and
+            will lead to a new generated file called '<asset_dir>/<name>.pb'.
+        proto_dep_name: str. The name of the proto library that contains the proto
             definition being converted to binary.
-        proto_type_name: str. The name of the proto type being converted in the text proto. This is
-            assumed to be part of the shared 'model' package.
-        asset_dir: str. The path to the assets directory.
-        proto_dep_path: str. The path to the proto_dep.
-        proto_package: str. The name of the proto package.
+        proto_type_name: str. The name of the proto type being converted in the text proto.
+        asset_dir: str. The path to the assets directory where the textproto files are present.
+            Example: 'src/main/assets'.
+        proto_dep_path: str. The path to the library that contains the proto_dep.
```

Comment:
This additional documentation is really helpful. This actually seems to be a Bazel path prefix, so I suggest calling it proto_dep_bazel_target_prefix to clarify. We're overloading the term 'path' to mean both filesystem and Bazel paths, so we should try to be more specific.

Reply:
Done